helm/odigos/templates/odiglet/daemonset.yaml

```diff
@@ -88,6 +88,8 @@ spec:
                   fieldPath: metadata.namespace
             - name: OTEL_LOG_LEVEL
               value: info
+            - name: OTEL_GO_OFFSETS_FILE
+              value: /offsets/offset_results.json
```

Comment:
WDYT about wrapping the offsets related sections in helm with `{{- if .Values.onPremToken }}`?